share/completions/apm.fish

```diff
@@ -40,7 +36,11 @@ function __fish_apm_using_command
                         return 1
                     end
                 case '*'
-                    return 1
+                    if test $command_seen_once -eq 0
+                        continue
```

Comment:
This way it still does some limited validation on the rest of the input (essentially checking that there's no command there twice). Even though there's only a small benefit it seemed to make sense to still do it. I can change it to a return statement though if you feel it'd be better.